src/Backend/Teradata/Helper/DropTableTrait.php

```diff
@@ -20,7 +20,8 @@ public function dropTableIfExists(string $dbName, string $tableName): void
 
     protected function tableExists(string $dbName, string $tableName): bool
     {
+        /** @var string $tableExists */
         $tableExists = $this->connection->fetchOne((new SqlBuilder())->getTableExistsCommand($dbName, $tableName));
-        return $tableExists !== false;
+        return $tableExists > 0;
```

Comment:
divny`string > 0`, ale funguje to 🤷

Reply:
jo, tam jde o to, ze ten fetchOne vraci mixed a to se nelibilo stanu